enterprise_gateway/services/processproxies/container.py

```diff
@@ -175,6 +175,7 @@ def kill(self):
         result = None
 
         if self.container_name:  # We only have something to terminate if we have a name
+            self.log.info(f"Killing Kernel with name: {self.container_name}")
```

Comment:
info -> debug
```suggestion
            self.log.debug(f"Terminating kernel with container name: {self.container_name}")
```
or replace with kernel_id...
```
self.log.debug(f"Terminating kernel: {self.kernel_id}")
```
although using the container name here is probably more appropriate and will contain the kernel_id.